apps/studio/components/interfaces/Database/Schemas/SchemaGraph.tsx

```diff
@@ -20,11 +20,15 @@ import { TableNode } from './SchemaTableNode'
 
 // [Joshen] Persisting logic: Only save positions to local storage WHEN a node is moved OR when explicitly clicked to reset layout
 
-export const SchemaGraph = () => {
-  const { ref } = useParams()
+export const SchemaGraph = ({ hideSchemaSelection = false }: { hideSchemaSelection?: boolean }) => {
+  const { ref, schema: selectedSchemaParam } = useParams()
   const { resolvedTheme } = useTheme()
   const { project } = useProjectContext()
-  const [selectedSchema, setSelectedSchema] = useState<string>('public')
+  const [selectedSchema, setSelectedSchema] = useState<PostgresSchema['name']>(
+    selectedSchemaParam || 'public'
+  )
+
+  const targetSchema = selectedSchemaParam || selectedSchema
```

Comment:
`selectedSchema` will never get used here if `selectedSchemaParam` is there. Is that correct?

Reply:
reverted the file